packages/light-apps/src/IdentityCard/IdentityCard.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import BN from 'bn.js';
+
+import keyring from '@polkadot/ui-keyring';
+import { Address, AddressSummary, NavButton } from '@polkadot/ui-components';
+
+import { RouteComponentProps } from 'react-router-dom';
+
+import { StyledCard, CardHeader, CardContent } from './IdentityCard.styles';
+import { camlizeInclusiveFirst } from '../utils/camelize';
+
+interface Props extends RouteComponentProps {}
+
+type State = {
+  address?: string,
+  buttonText: string
+  name?: string
+};
+
+const PLACEHOLDER_ADDRESS = '5'.padEnd(16, 'x');
+// FIXME: balance should come from LightAPI HOC's observables
+export class IdentityCard extends React.Component<Props, State> {
+  state: State = {
+    address: PLACEHOLDER_ADDRESS,
+    buttonText: 'Transfer'
+  };
+
+  componentDidMount () {
+    const { history, match, location } = this.props;
+
+    // For Debugging purposes only
+    if (process.env.NODE_ENV !== 'production') {
+      // @ts-ignore
+      window.h = history;
```

Comment:
Even if it's only for debugging purposes, this is quite ugly. If you want to check the current location, `window.location` should do. If you want to check the props of this component, the React Dev tools extension should help.